### DISCUSSION TITLE
Download all attachments [Aug 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3295] [Discord Threads]
> 🙋‍♂️ **AlexU**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### No option from the UI. You may want to explore NocoDB REST APIs.  
#  
> 🙋‍♂️ **AlexU**  
#### Ok thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1008089294818246757)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
